todos-app/docker-compose.yaml

```diff
@@ -0,0 +1,36 @@
+services:
+  frontend:
+    build: ./frontend
+    image: todos-frontend
+    container_name: todos-frontend
+    ports: 
+      - 8080:8080
+    volumes:
+      - ./frontend:/app
+    depends_on:
+      - task-service
+
+  task-service:
+    build: ./backend/task_service
+    image: task-service
+    container_name: task-service
+    ports:
+      - 3000:3000
```

Comment:
It would be helpful for reviewers if the port bind could be set to something other than 3000!(also 3306). 

Because rakuma development env uses 3000 port so it cannot run in parallel😢 

Please change like this!
```yml
task-db:
  ports:
    - "3316:3306"

task-service:
  ports:
    - "3001:3000"
```

Reply:
Please refer to this(maybe you need to add chrome container to E2E) 👍 
https://github.com/Fablic/training/blob/develop/myapp/docker-compose.yml

Reply:
I'll add selenium to docker compose file if I decide to do E2E testing!